frontend/tests/BadgesTestCase.php

```diff
@@ -40,7 +40,7 @@ public function tearDown(): void {
     }
 
     public static function getSortedResults(string $query) {
-        /** @var list<array{user_id: int}> */
+        /** @var list<array{user_id: int|null}> */
```

Comment:
```suggestion
        /** @var list<array{user_id: int}> */
```

se puede mejor arreglar la consulta para que esto jamás sea nulo?